Misc/NEWS.d/next/IDLE/2019-01-04-19-14-29.bpo-35660.hMxI7N.rst

```diff
@@ -0,0 +1 @@
+Remove use of `from tkinter import *` from window.py.
```

Comment:
Use double backquotes (though I doubt a news entry is wanted for such a change).

Reply:
I am not a fan of markup in blurbs, which get copied to idlelib NEWS, and I don't this it is needed here.

My understanding of the general policy is that a) if a change is non-trivial, there should be an issue, and b) if there is a non-trivial issue, there should be at least a short blurb.  To me, the only code change that is truly trivial is something that cannot affect users -- which is a change to comments.  Simple spelling or grammar changes to docstrings and docs also qualify.  I don't see anything in the devguide specifically addressing these points.

This issue fixes a style violation and a bug.  Like any refactoring, it could cause a user-visible effect if not done right.  A needed name could have been omitted.  The sys import bug could have been missed (there is no test).  The audience for blurbs includes future maintainers, including me.  I occasionally use the IDLE blurb list to check on what has been when.  I agree that this is a borderline case, but the last point is why I decided to keep at least a short note here.

Reply:
I think that news entries are for users, and future maintainers can use the VCS log.

The `Lib/idlelib/window.py` file is an implementation detail, it should not be directly imported by user code.

Reply:
The log is not quite the same.  Anyway, there is nothing special about window.py.  With a couple of exceptions, there is no official support for importing any of idlelib files.

If you think my handling of news items (and possibly that of predecessors*) is and has been wrong, please open a pydev issue stating what you think the policy is, or should be.  You might include a review of a month or two of *all* blurbs stating which ones you think should be deleted.

* I looked at idlelib/NEWS2.txt to get some idea of what KBK did.  There are several 'cleanup X' issues.  Some have issue numbers, some not.  "cleanup TextView", #1718043, appears to be internal only.  "cleanup EditorWindow.close" sounds like it might be, but there is no issue.  The Sourceforge SVN/r##### links on issues are dead (404) for me.  I cannot find either patch in the git log.  Perhaps I am not using it correctly.